webapp/src/widgets/editable.tsx

```diff
@@ -115,9 +116,26 @@ export function useEditable(
     }
 }
 
+function borderWidth(style: CSSStyleDeclaration): number {
+    return (
+        parseInt(style.borderLeftWidth || '0', 10) +
+        parseInt(style.borderRightWidth || '0', 10)
+    )
+}
+
 const Editable = (props: EditableProps, ref: React.Ref<Focusable>): JSX.Element => {
     const elementRef = useRef<HTMLInputElement>(null)
     const elementProps = useEditable(props, ref, elementRef)
+
+    useLayoutEffect(() => {
+        if (props.autoExpand && elementRef.current) {
+            const input = elementRef.current
+            const computed = getComputedStyle(input)
+            input.style.width = 'auto'
+            input.style.width = `${input.scrollWidth + borderWidth(computed) + 1}px`
```

Comment:
`+ 1` here is required for workaround [this Chrome issue](https://bugs.chromium.org/p/chromium/issues/detail?id=980476).